Code/FooBar.php

```diff
@@ -6,19 +6,27 @@ class FooBar
 {
     public function checker($number)
     {
-        if (is_int($number) && $number > 0) {
-            if ($number % 3 == 0 && $number % 5 == 0) {
-                $result = "Foo, Bar";
-            } elseif ($number % 5 == 0) {
-                $result = "Bar";
-            } elseif ($number % 3 == 0) {
-                $result = "Foo";
-            } else {
-                $result = strval($number);
-            }
-            return $result;
-        } else {
+        $foo = "Foo";
```

Comment:
Ok, variable name is not perfect, but it's not the main problem. Let's say I want to change "3" to another number - I will need to do it in more than one place.